packages/core/test/validate_test.js

```diff
@@ -127,11 +128,18 @@ describe("Validation", () => {
         expect(errors[1].message).eql("should be string");
       });
 
+      // ajv@7.0^ has changed the format of error.dataPath. In ajv@6.0^ the error
+      // in the illformed key will result in dataPath = ['bar.\\'"[]()=+*&^%$#@!'].
+      // However in ajv@7.0 and higher this key results in
+      // dataPath = bar.'"[]()=+*&^%$#@!. lodash breaks this into error paths
```

Comment:
note to @epicfaace -- look into why this is happening

Reply:
@newt10 provide code sample to test this on ajv

Reply:
Here is a code sample -> https://runkit.com/embed/qtzmcsm9k62w

Reply:
@epicfaace Here is the code sample -> https://runkit.com/embed/qtzmcsm9k62w